pkg/agent/openflow/pipeline_windows.go

```diff
@@ -281,5 +281,25 @@ func (c *client) hostBridgeUplinkFlows(localSubnet net.IPNet, category cookie.Ca
 			Cookie(c.cookieAllocator.Request(category).Raw()).
 			Done(),
 	}
+	if c.encapMode.SupportsNoEncap() {
+		// The packages received from uplink are forwarded to OVS bridge local interface by default.
+		// If NoEncap is enabled, the reply packets from remote Pod can be forwarded to local Pod directly.
```

Comment:
You should update the comments?

Reply:
Thanks for reminding. I forgot to update old comment after appending new one. Updated.